Nautilus/Handlers/ModDatabankHandler.cs

```diff
@@ -0,0 +1,110 @@
+using BepInEx;
+using Nautilus.Utility;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Nautilus.Handlers;
+/// <summary>
+/// A handler class for adding databank entries for mods.
+/// </summary>
+public static class ModDatabankHandler
+{
+    private static bool isInit;
+    private static List<PDAEncyclopedia.EntryData> waitList = new();
+    internal static bool isEnabled = true;
+    internal static void Initialize(uGUI_EncyclopediaTab tab)
+    {
+        LanguageHandler.SetLanguageLine("EncyPath_Mods", "Mods");
+        isinit = true;
```

Comment:
Compiler error because the field was renamed

Reply:
ah.